comm/src/main/scala/coop/rchain/comm/transport/GrpcTransportServer.scala

```diff
@@ -80,7 +79,10 @@ class GrpcTransportServer(
   ): Task[Cancelable] = {
 
     val dispatchSend: Send => Task[Unit] =
-      s => dispatch(s.msg).attemptAndLog >> metrics.incrementCounter("dispatched.messages")
+      s =>
+        dispatch(s.msg)
+          .executeWithOptions(TaskContrib.enableTracing(tracing))
+          .attemptAndLog >> metrics.incrementCounter("dispatched.messages")
 
     val dispatchBlob: StreamMessage => Task[Unit] =
```

Comment:
Don't you have to change `dispatchBlob` aswell?

Reply:
you sir are right. weird that it did not blow up in testing.